src/components/AFEpage/styles.js

```diff
@@ -0,0 +1,37 @@
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles((theme) => ({
+  Grid_Space: {
+    margin: "30px 0px 30px 0px",
+  },
+  AFE_approaches: {
+    fontWeight: 700,
+  },
+  Approaches_text: {
+    marginLeft: "28px",
+  },
+  Hadings: {
+    marginBottom: "20px",
+  },
+
+  Fiber_space: {
+    marginRight: "20px",
+  },
+  Ng_AFE: {
+    marginBottom: "50px",
+  },
+  Img_size: {
+    width: 328,
+    height: 210,
+  },
+  Large_img_size: {
+    width: 543,
+    height: 360,
+  },
+  IMG: {
+    width: "95%",
+    height: "90%",
+  },
```

Comment:
If images are scaled up, they can look grainy.  Also by setting the width and height to be exact percentages, these images can become scaled out of proportion depending on the container size.  Perhaps something like this would accomplish what you want:
```suggestion
  IMG: {
    max-width: "95%",
    height: "auto",
    padding: "5% 0",
  },
```